manifests/w/wenming/ThirdSpace/0.2.0/wenming.ThirdSpace.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: wenming.ThirdSpace
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: WenMing
```

```suggestion
Publisher: thirdspace
```
This should match what is written to control panel, or AppsAndFeaturesEntries should be specified in the installer manifest